src/ssaver.h

```diff
@@ -17,10 +17,10 @@
 
 typedef struct _ksaver_data {
 
-  const char *display_name;           // Usually this can be set to NULL to attach to first available display
-  unsigned long idle_timeout;         // seconds to idle before starting the screen saver
-  const char *saver_program;          // path to binary program that paints the screen saver
-  const char *saver_hooks;            // path to a hook script that will be executed to alert on screen saver transitions (start, finish)
+  char *display_name;           // Usually this can be set to NULL to attach to first available display
+  unsigned long idle_timeout;   // seconds to idle before starting the screen saver
+  char *saver_program;          // path to binary program that paints the screen saver
+  char *saver_hooks;            // path to a hook script that will be executed to alert on screen saver transitions (start, finish)
```

Comment:
Why can't we just make this a `std::string` and get rid of having to manage any of the memory and so don't have any problems with leaks. `Configuration::get_config_string()` already returns a `std::string` so that bit becomes easier and we just need to modify [`setup_ssaver()`](https://github.com/KanoComputing/kdesk/blob/master/src/ssaver.cpp#L97)/[`idle_time`](https://github.com/KanoComputing/kdesk/blob/master/src/ssaver.cpp#L120) to call `.c_str()` on it instead.